app/src/main/java/edu/uc/hickmadc/customnotifications/MainActivity.kt

```diff
@@ -1,18 +1,15 @@
 package edu.uc.hickmadc.customnotifications
 
-import android.annotation.SuppressLint
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
-import android.widget.Button
 import android.widget.ImageButton
 import android.widget.ListView
-import android.widget.Toast
 import androidx.navigation.Navigation
 import androidx.navigation.findNavController
 import androidx.navigation.ui.NavigationUI
 import com.google.android.material.bottomnavigation.BottomNavigationView
-import edu.uc.hickmadc.customnotifications.ui.main.MyListAdapter
-import edu.uc.hickmadc.customnotifications.ui.main.dialogfragment
+import edu.uc.hickmadc.customnotifications.ui.main.adapters.MyListAdapter
```

Comment:
Suggestion 1: I saw that the team had moved one of their adapters to its own "adapter" package, but it wasn't within the UI package. I figured that the number of UI classes was starting to get a bit unwieldy, so I organized them into "adapter" and "fragments" packages.
Source: https://kotlinlang.org/docs/coding-conventions.html#directory-structure

Reply:
Hmmmm, ok.